### ISSUE TITLE
Bug: Crashes on 1.15.2 Forge with Immersive Portals

### ISSUE THREAD

**Issue by 13rac1** [13rac1/Minecraft-Water-Erosion#10]
[Two Curseforge users (davdthethird, vhslance) reported a Minecraft startup crash when combined with Immersive Portals.](https://www.curseforge.com/minecraft/mc-mods/water-erosion?comment=15) Configuration:
> minecraft 1.15.2 and forge 31.1.0
> immersive-portals-0.4(for+mc+1.15+with+forge)
> MixinBootstrap-1.0.x(tested with 1.0.0 / 1.0.1 / 1.0.2)
> water-erosion-1.15.2-forge-1.0.1

```
--
 ERROR
 Game crashed
 An unexpected issue occurred and the game has crashed. We're sorry for the inconvenience.
--
```

Definitely unexpected. I don't see how these mods could be interacting. Initial guess is the SDK translation interface layer.

**Comment by 13rac1** [comment]
Running in dev the logged Fatal error:
```
[main/FATAL] [mixin/]: Mixin apply failed assets/immersive_portals/immersive_portals.mixins.json:MixinEntity -> net.minecraft.entity.Entity: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method func_174813_aQ in assets/immersive_portals/immersive_portals.mixins.json:MixinEntity was not located in the target class net.minecraft.entity.Entity. Using refmap main.refmap.json
```

The MCPBot CSV says `func_174813_aQ` is `getBoundingBox()` which is a valid function on Entity.

**Comment by LXGaming** [comment]
Hello, I was made aware of your mod by this issue https://github.com/LXGaming/MixinBootstrap/issues/5

This issue here is caused by your mod `water-erosion` and `immersive-portals` having the same refmap name `main.refmap.json`, whats happening is your refmap is getting overwritten by `immersive-portals` which is then obviously causing major issues.

The fix for this is to simply rename your refmap so that it contains your mod id. Example: `erosion.refmap.json`, this needs to be changed in your `build.gradle` and the `refmap` entry in `erosion.mixins.json`

**Comment by 13rac1** [comment]
Ah ha! Thanks! This is my first Minecraft mod, so I didn't realize my `refmap` would end up in the same place as others. I was testing this as the solution last night. I'll release a new version right now.

**Comment by 13rac1** [comment]
Fixed in `1.0.2` and `1.1.0-alpha2`.
